docs/en/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD.md

```diff
@@ -136,6 +136,14 @@ Parameter introduction:
 
 21. send_batch_parallelism: Integer, used to set the parallelism of sending batch data. If the value of parallelism exceeds `max_send_batch_parallelism_per_job` in the BE configuration, the BE as a coordination point will use the value of `max_send_batch_parallelism_per_job`.
 
+22. hidden_columns: Specify hidden column when no `columns` in Headers，multi hidden column shoud be
+separated by commas。
+
+       ```
+           hidden_columns: __DORIS_DELETE_SIGN__
```

Comment:
Please also give the example for `__DORIS_SEQUENCE_COL__`.
Or use may not known the name of this hidden column.

And the following header will result in different:

```
hidden_columns: __DORIS_DELETE_SIGN__,__DORIS_SEQUENCE_COL__
hidden_columns: __DORIS_SEQUENCE_COL__,__DORIS_DELETE_SIGN__
```

My suggestion is:
1. User can specify the hidden columns in different order.
2. The system will reorder the hidden columns specified by user,  put `__DORIS_DELETE_SIGN__` at first and `__DORIS_SEQUENCE_COL__` at last.

Reply:
use the order specified by user is easy to understand